src/styles/global.ts

```diff
@@ -181,3 +190,15 @@ export const LearningMaterialListItem = styled.li`
     }
   }
 `
+
+export const FlexContainer = styled.div`
+  display: flex;
+  justify-content: center;
+
+  gap: 8rem;
+
+  ${mq.sm} {
+    display: flex;
```

Comment:
don't need to repeat `display: flex`, right?

Reply:
Yes, you are absolutely right. I will fix it.